components/clp-package-utils/clp_package_utils/scripts/compress.py

```
@@ -56,7 +57,8 @@ def _generate_logs_list(
             return len(parsed_args.paths) != 0
 
         no_path_found = True
-        with open(host_logs_list_path, "r") as host_logs_list_file:
+        resolved_host_logs_list_path = resolve_host_path(pathlib.Path(host_logs_list_path))
+        with open(resolved_host_logs_list_path, "r") as host_logs_list_file:
```

Ruff (0.14.2)

Unnecessary mode argument

Remove mode argument

(UP015)